scripts/audit-plugin.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+#
+# Copyright (c) 2019-2021, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This script analyzes the hashCommitsWithMessages.log generated by scripts/audit-spark*.sh
+# and determines which commits are directly modifying classes that we use in our plugin. Commits
```

Comment:
what does "classes that we use in our plugin". mean?  There are different types of things we do.
1) directly copy classes and functions (ie functions aren't classes and we may have moved to other classes).
2) We have classes very similar ie cpu SortExec => GpuSortExec and we may have functionality very similar in there.

it would be nice to clarify exactly what this script does.

Reply:
this script will see if we have a dependency on that class that was modified in the spark commit. I will clarify it in the comment

Reply:
what do you mean by dependency?  we call it or import it?  This would all be good information to add to the description as well.